ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/checkboxgroup.html

```diff
@@ -38,8 +38,8 @@
             <div class="cmp-adaptiveform-checkboxgroup-item ${checkboxgroup.name}">
                 <label class="cmp-adaptiveform-checkboxgroup__label" aria-label="${checkboxgroup.enumNames[itemList.index]}"
                        title="${checkboxgroup.tooltipVisible ? '' : checkboxgroup.tooltipText}"
-                       for="${component.id}${'_'}${itemList.index}__widget">
-                    <input class="cmp-adaptiveform-checkboxgroup__option__widget" type="checkbox" id="${component.id}${'_'}${itemList.index}__widget"
+                       for="${component.id}__widget${'_'}${itemList.index}">
```

Comment:
we don't need a for attribute in this label as this already encloses the input tag. so we can get rid of the id here.

Reply:
yes, removed this and verified accessibility doesn't drop